lib/logstash/outputs/scalyr.rb

```diff
@@ -661,6 +661,17 @@ def build_multi_event_request_array(logstash_events)
           ).force_encoding('UTF-8')
         end
         event_json = scalyr_event.to_json
+      rescue Java::JavaLang::ClassCastException => e
+        # Most likely we ran into the issue described here: https://github.com/flori/json/issues/336
+        # Because of the version of jruby logstash works with we don't have the option to just update this away,
+        # so if we run into it we convert bignums into strings so we can get the data in at least.
+        # This is fixed in JRuby 9.2.7, which includes json 2.2.0
+        Scalyr::Common::Util.convert_bignums(scalyr_event)
```

Comment:
Let's also log a message here when this happens and add another metric (monotonically increasing counter) so we will have some idea how often that happens.